src/main/java/duke/Duke.java

```diff
@@ -0,0 +1,100 @@
+package duke;
+
+import duke.task.Deadline;
+import duke.task.Event;
+import duke.task.Task;
+import duke.task.ToDo;
+
+import java.util.ArrayList;
+import java.io.IOException;
+
+public class Duke {
+    private static ArrayList<Task> taskList = new ArrayList<Task>();
+
+    public static void main(String[] args) {
+        FileProcessor fileProcessor = null;
+        try {
+            fileProcessor = new FileProcessor(taskList);
+        } catch (IOException e) {
+            System.out.println("Error");
+        }
+        Ui ui = new Ui();
+        Parser parser = new Parser();
+        ui.greet();
+        String input = ui.getUserInput();
+        while (input.compareTo("bye") != 0) {
+            String[] parsedInput;
+            try {
+                parsedInput = parser.parseInput(input);
+            } catch (DukeException e) {
+                System.out.println("Oops, I'm sorry, I don't know what that means :(");
```

Comment:
Since you have a UI class, consider moving all UI operations to that class including all the lines that print to the console. This will help you avoid the use of magic literals as well. [See here](https://nus-cs2113-ay2223s2.github.io/website/se-book-adapted/chapters/codeQuality.html#avoid-magic-numbers)